llvm/test/MC/RISCV/corev/hwlp-all-extensions.s

```diff
@@ -1,6 +1,8 @@
 # RUN: llvm-mc -triple=riscv32 --mattr=xcorev -show-encoding %s 2>&1 \
```

Comment:
Should this say +xcorev to match the other tests where --mattr is provided +extension